### DISCUSSION TITLE
Discussion on sliderend missing animation in osu lazer

### DISCUSSION THREAD

**Discussion by ProMedic4U** [ppy/osu#26653] [Ideas] [answered]
The first thing is that the animation is too long. I 100% agree with adding sliderend missing indication since it is now skinnable. But I find the default animation really distracting especially on slider-heavy maps, because the animation is way too long.
Secondly, I do not know the point of both showing miss animation and sliderend miss animation when not clicking at the slider head. Like, why would I care about slider end if I already missed the whole slider? The following sliderend miss animation is not necessary anyway. It just adds more distraction.
All I want is cleaner gameplay from lazer. That is the whole point of this post.

**Comment by peppy** [comment]
> Secondly, I do not know the point of both showing miss animation and sliderend miss animation when not clicking at the slider head

The tail is worth 150 in addition to the head. So they both show misses.

I am in agreement with changing the animation a bit.

**Reply by peppy** [reply, marked as answer]
@ProMedic4U I've reduced the animation in https://github.com/ppy/osu/pull/26668. Hopefully this will help towards acceptance.

**Comment by mixalisnew** [comment]
i agree with that when i tried lazer a few days ago with the skin i use slider misses were noticeable to a level more than i would like. I dont have a specific suggestion on how to improve it but it needs some work

**Reply by peppy** [reply]
See #26668.
